File: trunk/SSZS/src/ws/hoyland/sszs/ReconObject.java
package ws.hoyland.sszs;

public class ReconObject {
	
	private static ReconObject instance;
	
	private ReconObject(){
		
	}
	
	public static ReconObject getInstance(){
		if(instance==null){
			instance = new ReconObject();
		}
		return instance;
	}
}

File: sszs-sln/sszs/ReconObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ws.hoyland.sszs
{
    public class ReconObject
    {
        private static ReconObject instance;

        private ReconObject()
        {

        }

        public static ReconObject getInstance()
        {
            if (instance == null)
            {
                instance = new ReconObject();
            }
            return instance;
        }
    }
}
